torch/_inductor/graph.py

```diff
@@ -136,6 +136,9 @@ def supported_dtype_of_cpp_wrapper(dtype: torch.dtype, device_type: str) -> bool
             torch.int32,
             torch.int16,
             torch.int8,
+            torch.uint64,
```

Comment:
I think with all your fixes we may be able to completely remove this `supported_dtype_of_cpp_wrapper` check now.

Reply:
I'll try it and see what happens!

Reply:
It seems like it worked just fine!